doc/classes/Object.xml

```diff
@@ -212,9 +212,12 @@
 				Combined with [method _get] and [method _get_property_list], this method allows defining custom properties, which is particularly useful for editor plugins. Note that a property [i]must[/i] be present in [method get_property_list], otherwise this method will not be called.
 				[codeblocks]
 				[gdscript]
+				var internal_data = {}
 				func _set(property, value):
```

Comment:
Should be added to C# too.
```suggestion
				var internal_data = {}

				func _set(property, value):
```

Reply:
I added C#, but not sure if it's correct.